### ISSUE TITLE
KW.1.1+Orf1a:L590F, S:N30H(46 seqs, 8 countries)

### ISSUE THREAD

**Issue by xz-keg** [sars-cov-2-variants/lineage-proposals#1979]
KW.1.1+G2035T, T14602C, A21650C(Orf1a:L590F, S:N30H)

GISAID query: G2035T, T14602C, A21650C
No. of seqs: 33(Canada 5 Denmark 2 France 1 Japan 1 New Zealand 1 USA 10 China 13(8 different provinces))

First: EPI_ISL_19150743, 2024-4-4, China
Latest: EPI_ISL_19371286, 2024-8-15, USA

[usher](https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice315_genome_test_65875_6beb10.json?label=id:node_5435123)
![image](https://github.com/user-attachments/assets/1a3e10be-ed98-49ba-aca9-d6b66797a825)

**Comment by FedeGueli** [comment]
+2 in the Liaoning branch

**Comment by aviczhl2** [comment]
+1 Australia

**Comment by FedeGueli** [comment]
39 again Liaoning

**Comment by DailyCovidCases** [comment]
@FedeGueli This branch has slowed down a lot, please close this

**Comment by FedeGueli** [comment]
Emerged in an undersampled area. Liaoning usually uploads via batches we should wait the next one.

**Comment by xz-keg** [comment]
no new seqs

**Comment by FedeGueli** [comment]
it reappeared suddenly in Ontario with 3 recent samples all with S:N237S and S:K554N additonal spike mutations. reopening for a while just to be sure it doesnt spread further

**Comment by xz-keg** [comment]
+2 Yunnan but old seqs

**Comment by FedeGueli** [comment]
> but old seqs

Still interesting to notice that while worlwide N30 turned to negative N Glycan , in China there were an opposite evolutive pressure pushing toward positive charged K or H

**Comment by xz-keg** [comment]
> Still interesting to notice that while worlwide N30 turned to negative N Glycan , in China there were an opposite evolutive pressure pushing toward positive charged K or H

yeah but only for the N30 glycan, T22N seems not affected. China seems never had some significant independent emergences of 31del or 32del despite we know it shall not be rare.

**Comment by FedeGueli** [comment]
no recent samples closing it

**Comment by FedeGueli** [comment]
dead
